src/main/java/rs/controller/MyTenantController.java

```diff
@@ -0,0 +1,28 @@
+package rs.controller;
+
+import lombok.extern.log4j.Log4j2;
+import org.folio.tenant.rest.resource.TenantApi;
+
+// an example how to override tenant controller
+// tne name of the bean MUST be folioTenantController see org.folio.spring.controller.TenantController for details
+@Log4j2
+//@RestController("folioTenantController")
```

Comment:
Why the controller is commented out? And why the name of the bean is different from id?